spec/javascripts/packages/document-capture/higher-order/with-props-spec.jsx

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react';
+import withProps from '../../../../../app/javascript/packages/document-capture/higher-order/with-props';
```

Comment:
Conventionally we use the NPM scope to save from having to traverse so far back up the file tree:

```suggestion
import withProps from '@18f/identity-document-capture/higher-order/with-props';
```

Reply:
Heh, you beat me by about 30 seconds with aae3d48 😄

Reply:
Yeah, I saw that. I fixed it in the most recent commit.